exercises/rotational-cipher/RotationalCipherTest.cs

```diff
@@ -1,25 +1,25 @@
-// This file was auto-generated based on version 1.0.0 of the canonical data.
+// This file was auto-generated based on version 1.1.0 of the canonical data.
```

Comment:
It's cool to see this get updated as the canonical data got updated and to know that it was done problematically... your hard work is paying off @ErikSchierboom :)